src/prompts/password.rs

```diff
@@ -73,6 +77,9 @@ pub struct Password<'a> {
     /// Message to be presented to the user.
     pub message: &'a str,
 
+    /// Message to be presented to the user when verifying the input.
+    pub verification_message: &'a str,
```

Comment:
Wdyt of using the `confirmation` terminology? It is the term that I see on Web UX resources, I'm not sure if the convention/history for CLIs are different.

Reply:
Yes, this suits much better I think; I couldn't think of the term at the moment of writing this PR 😅 I will push a fix for changing the terminology.